src/configs/rust-in-replit.yml

```diff
@@ -8,13 +8,13 @@
 files:
   [
     {
-      "source": "/tooling/locales/en/*.md",
-      "dest": "/rust-in-replit/tooling/locales/en/%original_file_name%",
-      "translation": "/tooling/locales/%two_letters_code%/%original_file_name%",
+      "source": "/tooling/locales/english/*.md",
+      "dest": "/rust-in-replit/tooling/locales/english/%original_file_name%",
+      "translation": "/tooling/locales/%language%/%original_file_name%",
```

Comment:
```suggestion
      "source": "/tooling/locales/english/*.md",
      "dest": "/rust-in-replit/tooling/locales/en/%original_file_name%",
      "translation": "/tooling/locales/%language%/%original_file_name%",
```

Reply:
@raisedadead Requested this change

Reply:
It would bring it into parity with our other projects - as long as we're fine reapplying/reapproving translations we're good.

Reply:
That's the idea.